RNTester/RNTester/AppDelegate.mm

```diff
@@ -161,6 +161,15 @@ - (void)loadSourceForBridge:(RCTBridge *)bridge
   _turboModuleManager = [[RCTTurboModuleManager alloc] initWithBridge:bridge
                                                              delegate:self
                                                             jsInvoker:bridge.jsCallInvoker];
+
+#if RCT_DEV
+  /**
+   * Eagerly initialize RCTDevMenu so CMD + d, CMD + i, and CMD + r work.
+   * This is a stop gap until we have a system to eagerly init Turbo Modules.
+   */
+  [_turboModuleManager moduleForName:"RCTDevMenu"];
```

Comment:
We probably want this in `RNTester_macOS/AppDelegate.mm` too. As a rule of thumb, any new changes to an iOS file should default to be made in the macOS version of the same file unless there's something iOS specific about it

Reply:
Added.